library/core/src/panic.rs

```diff
@@ -333,6 +376,17 @@ impl fmt::Display for Location<'_> {
     }
 }
 
+#[unstable(
+    feature = "panic_internals",
+    reason = "internal details of the implementation of the `panic!` and related macros",
+    issue = "none"
+)]
+#[doc(hidden)]
+#[derive(Debug, Copy, Clone)]
+pub enum ExtraInfo<'a> {
```

Comment:
You can use `#[non_exhaustive]` to state that this enum might get more members in the future.
```suggestion
#[non_exhaustive]
pub enum ExtraInfo<'a> {
```